docs/advanced.rst

```diff
@@ -127,6 +127,29 @@ referencing the ``changed_by`` field:
 
 Admin integration requires that you use a ``_history_user.setter`` attribute with your custom ``_history_user`` property (see :ref:`admin_integration`).
 
+Custom ``history_id``
+---------------------
+
+If you want to use something other then an ``AutoField`` for the history
+table primary key you can pass in a ``history_id_field``.  A common use case
+for this would be to use a ``UUIDField``.
+
+For any field provided it will automatically set ``primary_key=True`` and
+``editable=False``.
```

Comment:
"Note: regardless of what field type you specify as your `history_id` field, that field will automatically set ``primary_key=True`` and ``editable=False``."